tests/test_list.py

```
@@ -440,12 +440,30 @@ def test_list_archives_bad_xml(monkeypatch, xml_content: str):
 def test_tool_modules(monkeypatch, host: str, target: str, tool_name: str):
     archive_id = ArchiveId("tools", host, target)
-    in_file = "{}-{}-{}-update.xml".format(host, target, tool_name)
     expect_out_file = "{}-{}-{}-expect.json".format(host, target, tool_name)
-    _xml = (Path(__file__).parent / "data" / in_file).read_text("utf-8")
     expect = json.loads((Path(__file__).parent / "data" / expect_out_file).read_text("utf-8"))
 
-    monkeypatch.setattr(MetadataFactory, "fetch_http", lambda self, _: _xml)
+    if tool_name != "tools_ifw":
+        in_file = "{}-{}-{}-update.xml".format(host, target, tool_name)
+        _xml = (Path(__file__).parent / "data" / in_file).read_text("utf-8")
+        _mock_fetch_http = lambda self, _: _xml
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)